frontend/src/concepts/areas/types.ts

```diff
@@ -72,6 +78,12 @@ export enum StackCapability {
   SERVICE_MESH_AUTHZ = 'CapabilityServiceMeshAuthorization',
 }
 
+export type CustomConditionFunction = (state: {
```

Comment:
Can you add here a comment explaining what's a Custom Condition? use the same convention as the attributes we have in `SupportedComponentFlagValue` but we should be really clear about the usage of external logic such as dsc/dsci manipulation